src/helpers.js

```diff
@@ -228,6 +228,38 @@ function checkSanitizeDeprecation(opt) {
   }
 }
 
+function indentCodeCompensation(raw, text) {
+  if (raw === undefined) {
+    return text;
+  }
+
+  const matchIndentToCode = raw.match(/^(\s+)(?:```)/);
+
+  if (matchIndentToCode === null) {
+    return text;
+  }
+
+  const [indentToCode] = matchIndentToCode.reverse();
```

Comment:
```suggestion
  const indentToCode = matchIndentToCode[matchIndentToCode.length - 1];
```

No need to iterate over each item in the array if you only need the last one.